Sources/Quick/DSL/DSLValidation.swift

```diff
@@ -0,0 +1,28 @@
+/**
+    Descriptors intended to identify DSL function context.
+*/
+public enum ExampleContextDescriptor: String {
+    case None = "none"
```

Comment:
It's not used, and in the code there is no need for a "default" value like this. It will be removed.